feature/topics/src/main/kotlin/uk/govuk/app/topics/TopicUiState.kt

```diff
@@ -0,0 +1,14 @@
+package uk.govuk.app.topics
+
+import uk.govuk.app.topics.ui.model.TopicUi
+
+internal sealed class TopicUiState(
+    val topicUi: TopicUi? = null,
```

Comment:
I don't think we need this in here as it is only relevant in the default state

Reply:
Done